distributor-api-v1/operations.md

```diff
@@ -872,16 +872,17 @@ Gives a pricing information for the given configuration.
 | `PostalCode` | string | optional | Postal code of the address. |
 | `StateCode` | string | optional | ISO 3166-2 code of the state, e.g.`US-AL`. |
 | `NationalityCode` | string | optional | ISO 3166-1 Aplha-2 code of the customer’s nation country, e.g.`US`. |
-| `SendMarketingEmail` | boolean | optional | Subscribe to marketing emails |
+| `SendMarketingEmail` | boolean | optional | Subscribe to marketing emails. When booker is present, this should always be false or null. |
 
-#### Booker   <a id="booker"></a>
+#### Booker
 
 |  | Property | Type | Description |
 | :--- | :--- | :--- | :--- |
 | `Email` | string | required | Email of the booker. |
 | `FirstName` | string | required | First name of the booker. |
 | `LastName` | string | required | Last name of the booker. |
 | `Telephone` | string | optional | Telephone number of the booker. |
+| `SendMarketingEmail` | boolean | optional | Subscribe to marketing emails. When booking for someone else, this should have the value, not [Customer](operations.md#customer). |
```

Comment:
Maybe mention something like `When not provided, subscription is disabled`? If it's what BE does.